docs/cody/proxy-setup.mdx

```diff
@@ -0,0 +1,112 @@
+
+# Proxy Setup Guide
+
```

Comment:
@MaedahBatool Let's figure out where the placement of these docs makes the most sense

Reply:
@aramaraju I agree we need to move it around to some better place.